app/helpers/data_getter.py

```diff
@@ -733,12 +734,12 @@ def get_locations_of_events():
             return names
 
     @staticmethod
-    def get_sales_open_tickets(event_id, give_all=False):
+    def get_sales_open_tickets(event_id, event_timezone='UTC', give_all=False):
         if give_all:
             tickets = Ticket.query.filter(Ticket.event_id == event_id)
```

Comment:
Please remove `give_all` as no function call is using it atm and also it does not work. Also modify all the function calls accordingly.